### PR TITLE
throw an error when sabotage is detected (instead of unpacking last .tlf...

### PR DESCRIPTION
**Pull request by domsj**
.../.tlx)

### CODE REVIEW

File: extension/test/server/right/system_tests_long_right.py [hunk 1]
```diff
@@ -487,18 +487,12 @@ def test_sabotage():
...
-    Common.iterate_n_times(2000, Common.simple_set)
-    logging.info("sleeping fo 10s")
-    time.sleep(10)
 
-    size = q.system.fs.fileSize("%s/001.tlf" % node_tlf_dir)
-    logging.info("file_size = %i", size)
-    assert_true(size > 1024 * 5)
+    cmd = []
+    command = clu._cmd('sturdy_0')
+    cmd.extend(command)
```
Comment (toolslive) [inline extension/test/server/right/system_tests_long_right.py]: cmd = clu._cmd('sturdy_0')

File: src/node/node_main.ml [hunk 2]
```diff
@@ -801,6 +801,10 @@ let _main_2 (type s)
             let rc = 49 in
             Logger.fatal_f_ "[rc=%i] Tlog has a checksum error, last valid pos = %Li" rc pos >>= fun () ->
             Lwt.return rc
+          | Tlogcommon.TLogSabotage ->
+            let rc = 50 in
+            Logger.fatal_f_ "[rc=%i] Somebody has been messing with the available tlogs" rc >>= fun () ->
+            Lwt.return rc
```
Comment (toolslive) [inline src/node/node_main.ml:807]: I just wonder how long it will take before this gets reported back to us ;)

**Comment by toolslive** [comment]
tentative LGTM

File: src/node/node_main.ml (hunk 2 above)
Comment (domsj) [inline src/node/node_main.ml:807]: Hehe, I wonder the same thing. But it should definitely make them think a little about what they're doing before they harass us :-)

**Pull request merged**